libs/PiwikTracker/PiwikTracker.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+// PiwikTracker.php is now managed by composer. To prevent breaking existing
+// code, this file has been left as a redirect to its new location in the
+// vendor directory.
+if (!class_exists('PiwikTracker')) {
+    require_once __DIR__ . '/../../vendor/piwik/piwik-php-tracker/PiwikTracker.php';
```

Comment:
To play it safe, and prevent issue in case someone changes the constraint in composer.json to later include eg. 2.0 release, could you here add a sanity check eg. 
```php
if(PiwikTracker::VERSION !== 1) { 
    throw new Exception("Expected PiwikTracker in libs/PiwikTracker/PiwikTracker.php to be version 1 for keeping backward compatibility. ");
}

```